iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/AreaSerDeTest.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.serde;
+
+import com.powsybl.iidm.network.*;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+
+import static com.powsybl.iidm.serde.IidmSerDeConstants.CURRENT_IIDM_VERSION;
+
+/**
+ * @author Marine Guibert {@literal <marine.guibert at artelys.com>}
+ * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
+ */
+class AreaSerDeTest extends AbstractIidmSerDeTest {
+
+    @Test
+    void testNetworkAreas() throws IOException {
+        Network network = createBaseNetworkWithAreas();
+        allFormatsRoundTripTest(network, "/areaRoundTripRef.xml", CURRENT_IIDM_VERSION);
```

Comment:
Could you also add:
```java
allFormatsRoundTripTest(network, "/areaRoundTripRef.xml", IidmVersion.V_1_12);
```
to check that the network is also exportable with a prior version.

(This is not checked by the following assertion.)

Reply:
adressed in 77ae1f322f10dae4af6d17219c9a4586e05ca194